worlds/kh1/Options.py

```diff
@@ -0,0 +1,325 @@
+    display_name = "Puppies"
+    option_full = 0
+    option_triplets = 1
+    option_individual = 2
+    default = 1
+
+class EXPMultiplier(NamedRange):
+    """
+    Determines the multiplier to apply to EXP gained
+    """
+    display_name = "EXP Multiplier"
+    default = 16
+    range_start = default / 4
+    range_end = 128
+    special_range_names = {
+        "0.25x": default / 4,
+        "0.5x": default / 2,
+        "1x": default,
+        "2x": default * 2,
+        "3x": default * 3,
+        "4x": default * 4,
+        "8x": default * 8,
+    }
+
+class RequiredReportsEotW(Range):
+    """
+    If End of the World Unlock is set to "Reports", determines the number of Ansem's Reports required to open End of the World.
+    """
+    display_name = "Reports to Open End of the World"
+    default = 4
+    range_start = 0
+    range_end = 13
+    
+class RequiredReportsDoor(Range):
+    """
+    If Final Rest Door is set to "Reports", determines the number of Ansem's Reports required to manifest the door in Final Rest to challenge Ansem.
+    """
+    display_name = "Reports to Open Final Rest Door"
+    default = 4
+    range_start = 0
+    range_end = 13
+
+class ReportsInPool(Range):
+    """
+    Determines the number of Ansem's Reports in the item pool.
+    """
+    display_name = "Reports in Pool"
+    default = 4
+    range_start = 0
+    range_end = 13
+
+class RandomizeKeybladeStats(DefaultOnToggle):
+    """
+    Determines whether Keyblade stats should be randomized
+    """
+    display_name = "Randomize Keyblade Stats"
+
+class KeybladeMinStrength(Range):
+    """
+    Determines the lowest STR bonus a keyblade can have
+    """
+    display_name = "Keyblade Minimum STR Bonus"
+    default = 3
+    range_start = 0
+    range_end = 20
+
+class KeybladeMaxStrength(Range):
+    """
+    Determines the maximum STR bonus a keyblade can have
+    """
+    display_name = "Keyblade Maximum STR Bonus"
+    default = 14
+    range_start = 0
+    range_end = 20
+
+class KeybladeMinMP(Range):
+    """
+    Determines the minimum MP bonus a keyblade can have
+    """
+    display_name = "Keyblade Minimum MP Bonus"
+    default = -2
+    range_start = -2
+    range_end = 5
+
+class KeybladeMaxMP(Range):
+    """
+    Determines the maximum MP bonus a keyblade can have
+    """
+    display_name = "Keyblade Maximum MP Bonus"
+    default = 3
+    range_start = -2
+    range_end = 5
+
+class LevelChecks(Range):
+    """
+    Determines the maximum level for which checks can be obtained.
+    """
+    display_name = "Level Checks"
+    default = 100
+    range_start = 0
+    range_end = 100
+
+class ForceStatsOnLevels(NamedRange):
+    """
+    If this value is less than the value for Level Checks, this determines the minimum level from which only stat ups are obtained at level up locations
+    For example, if you want to be able to find any multiworld item from levels 1-50, then just stat ups from level 51 on, set this value to 51.
```

Comment:
```suggestion
    If this value is less than the value for Level Checks, this determines the minimum level from which only stat ups are obtained at level up locations
    For example, if you want to be able to find any multiworld item from levels 1-50, then just stat ups for levels 51-100, set this value to 51.
```

I had to reread the new description a few times to figure out what it meant, so here's a suggestion on rewording it slightly

Reply:
Thanks!